maquetado/index.html

```diff
@@ -11,7 +11,7 @@
 <body>
   <nav class="nav">
     <img class="nav-logo" src="images/wolox_logo.svg" alt="Wolox's Logo">
-    <h2 class="nav-item">Developer Tools</h2>
+    <h2 class="nav-item dev-tools">Developer Tools</h2>
```

Comment:
Why you include this class?

Reply:
I added that class, for hiding in small views (< `1024px)